pkg/cmd/roachtest/acceptance.go

```diff
@@ -33,7 +37,7 @@ func registerAcceptance(r *registry) {
 		{"bank/cluster-recovery", runBankClusterRecovery},
 		{"bank/node-restart", runBankNodeRestart},
 		{"bank/zerosum-splits", runBankNodeZeroSum},
-		{"bank/zerosum-restart", runBankZeroSumRestart},
+		// {"bank/zerosum-restart", runBankZeroSumRestart},
```

Comment:
wait - why comment this out instead of putting a `t.Skip()` at the beginning of `runBankZeroSumRestart()` ?